ggml-rpc.h

```diff
@@ -10,12 +10,14 @@ extern "C" {
 #define GGML_RPC_MAX_SERVERS       16
 
 // backend API
-GGML_API GGML_CALL ggml_backend_t ggml_backend_rpc_init(const char * endpoint);
+GGML_API GGML_CALL void ggml_backend_rpc_setdevice(const char * endpoint, int device);
```

Comment:
I think it is a bad idea to add this as global state of the RPC backend. For example, this will prevent loading a model with a set of RPC servers, and a different model with a different set of RPC servers at the same time. Is it really necessary? I think it was ok to refer to each RPC server by a endpoint string, rather than a device index.

Reply:
The intent was to make it consistent with other backends like CUDA and sycl but I agree that having a global state is not great. I will add more special handling in `llama.cpp` and revert the changes in the RPC backend.